sell-loot.lic

```diff
@@ -19,16 +36,19 @@ class SellLoot
 
     @settings = get_settings
     town_data = get_data('town')
-    @character_hometown = @settings.hometown
-    @hometown = town_data[@settings.hometown]
+    @character_hometown = DRC.get_town_name(args.hometown || @settings.hometown)
+    @hometown = town_data[@character_hometown]
     @bankbot_name = @settings.bankbot_name
     @bankbot_room_id = @settings.bankbot_room_id
     @bankbot_deposit_threshold = @settings.bankbot_deposit_threshold
     @bankbot_enabled = @settings.bankbot_enabled
     @local_currency = town_data[@character_hometown]['currency']
     skip_bank = @settings.sell_loot_skip_bank
     skip_exchange = @settings.sell_loot_skip_exchange
-    keep_money_by_currency = @settings.sell_loot_money_on_hand
+    keep_money_by_currency = @settings.sell_loot_money_on_hand.split(' ')
+    keep_amount = args.amount || keep_money_by_currency[0] || 3
+    keep_denomination = args.type || keep_money_by_currency[1] || 'silver'
```

Comment:
The defaults of 3 silver was original logic; just moved here from the now-deleted `parse(..)` method